src/emulator.py

```diff
@@ -86,70 +88,57 @@ def get_status() -> dict:
 
 
 def start(version: str, wipe: bool, output_to_logfile: bool = True) -> None:
-    global proc
     global version_running
+    global EMULATOR
 
-    if proc is not None:
+    if EMULATOR is not None:
         log(
             f"Before starting a new emulator - version {version}, "
             f"killing the already running one - version {version_running}",
             "red",
         )
         stop()
 
-    path = ROOT_DIR / "src/binaries/firmware/bin"
+    # Conditionally redirecting the output to a logfile instead of terminal/stdout
+    if output_to_logfile:
+        logfile = open(helpers.EMU_BRIDGE_LOG, "a")
+    else:
+        logfile = sys.stdout
```

Comment:
Isn't this some bad copy&paste? Or should it really be here? I am just a bit confused it was not part of the previous PR

Reply:
It needs to be here now, as we are spawning emulator a different way. Previously it was with explicit `subprocess.Popen`, and when we do not supply `stdout` and `stderr` kwargs into it, it will automatically choose existing stdout and stderr.

However, current `LegacyEmulator` and `CoreEmulator` is _always_ calling the `stdout` and `stderr` arguments in `subprocess.Popen`, so we need to supply _either_ the logfile, or "current" stdout, otherwise the logfile will be set internally as `self.logfile = self.profile_dir / "trezor.log"`

Details at 
https://github.com/trezor/trezor-firmware/blob/30a77a76bc41296a32269694a666fadfc79d89ec/python/src/trezorlib/_internal/emulator.py#L69-L72
and
https://github.com/trezor/trezor-firmware/blob/30a77a76bc41296a32269694a666fadfc79d89ec/python/src/trezorlib/_internal/emulator.py#L132-L138